test/0.8.9/withdrawal-queue.test.js

```diff
@@ -33,7 +33,7 @@ contract('WithdrawalQueue', ([owner, stranger, daoAgent, user]) => {
     await steth.approve(withdrawalQueue.address, StETH(300), { from: user })
 
     await impersonate(ethers.provider, steth.address)
```

Comment:
maybe let's just use a separate address for finalization as I've done in `withdrawal-queue-share-rate-changes.test.js`? it's anyway role-based so using stETH mock doesn't add any value here